packages/babel-code-frame/src/index.ts

```diff
@@ -1,10 +1,19 @@
 import highlight, { shouldHighlight } from "@babel/highlight";
-import chalk, { type Chalk } from "chalk";
+
+import _chalk from "chalk";
+const chalk = _chalk as unknown as typeof import("chalk-BABEL_8_BREAKING-true");
+type Chalk =
+  typeof import("chalk-BABEL_8_BREAKING-true").Instance extends new () => infer R
+    ? R
+    : never;
```

Comment:
This is annoying, but there is no way to tell TS to resolve Chalk types from `chalk-BABEL_8_BREAKING-true` without breaking type checking (because some of our dependencies need types from other chalk versions). Also, Chalk 4 doesn't export this Chalk type so we have to extract it from `Instance`.